unicode/Makefile

```diff
@@ -1,6 +1,9 @@
-Scheme=../bin/scheme
+Scheme=../bin/zuo .. run
```

Comment:
Is that intended? It seems unrelated to unicode.

Reply:
Yes, there is no ../bin/scheme any more.